tests/async_tests/test_interfaces.py

```diff
@@ -50,6 +50,18 @@ async def test_https_request() -> None:
         assert len(http._connections[url[:3]]) == 1  # type: ignore
 
 
+@pytest.mark.usefixtures("async_environment")
+async def test_ftp_request() -> None:
```

Comment:
Don't think testing our behavior when requesting `ftp` is the primary goal of this test, so suggesting an alternative name?

```suggestion
async def test_request_unsupported_protocol() -> None:
```